### PR TITLE
print pfactor in log, fixes #130

### LINKED ISSUE #130: print pfactor for npt

### PR DESCRIPTION
**Pull request by alinelena**

### CODE REVIEW

File: janus_core/calculations/md.py [hunk 1]
```diff
@@ -692,9 +692,13 @@ def __init__(
...
 
         self.ttime = thermostat_time * units.fs
-        scaled_bulk_modulus = bulk_modulus * units.GPa
+
         if barostat_time:
-            pfactor = (barostat_time * units.fs) ** 2 * scaled_bulk_modulus
+            pfactor = (barostat_time * units.fs) ** 2 * bulk_modulus * units.GPa
+            if self.logger:
+                self.logger.info(
+                    "NPT pfactor=%s GPa fs^2", barostat_time**2 * bulk_modulus
```
Comment (ElliottKasoar) [inline janus_core/calculations/md.py:700]: Why not use `pfactor`? If it's avoiding the unit conversions, couldn't that be hiding any problems with the `pfactor` that's actually used?

Comment (alinelena) [reply janus_core/calculations/md.py]: done

**Review by ElliottKasoar — DISMISSED** [review]
Looks good to me (tests pass locally with the small MACE_MP model cached).

Only other question to consider is whether we might want this also/instead in the summary for the CLI

File: janus_core/calculations/md.py [hunk 2]
```diff
@@ -692,9 +692,12 @@ def __init__(
         super().__init__(ensemble=ensemble, file_prefix=file_prefix, *args, **kwargs)
 
         self.ttime = thermostat_time * units.fs
-        scaled_bulk_modulus = bulk_modulus * units.GPa
+
         if barostat_time:
-            pfactor = (barostat_time * units.fs) ** 2 * scaled_bulk_modulus
+            pfactor = barostat_time**2 * bulk_modulus
```
Comment (oerc0122) [inline janus_core/calculations/md.py:697]: Are `barostat_time` and `bulk_modulus` not already unitful? if not, why not?

Comment (oerc0122) [reply janus_core/calculations/md.py:697]: Ok, then a brief comment might want to be added on why.

**Review by ElliottKasoar — APPROVED** [review]

**Review by oerc0122 — APPROVED** [review]

**Pull request merged**